pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -57,6 +60,17 @@ def _apply_incubators(self):
             for egg in self.eggs:
                 if egg["used"] or egg["km"] == -1:
                     continue
+                
+                if self.breakable_incubator:
+                    if incubator.get('uses_remaining') is not None:
+                        if egg["km"] not in self.breakable_incubator:
+                            continue
+                    
+                if self.infinite_incubator:
+                    if incubator.get('uses_remaining') is None: 
```

Comment:
Ha actually I don't use incubator.get('uses_remaining') to get the number of use remaining, but to differentiate the infinite incubator from the other ones (that's the only way I know of). The infinite incubator send back "None" and the others send back "int".